ch5-catdir/controller.h

```diff
@@ -1,10 +1,22 @@
 #pragma once
 
-#include <cstdlib>
 #include <iosfwd>
 #include <span>
 #include <string_view>
 
+/*
+ * ExitStatus - constants for the exit status values returned by the program
+ */
+
+namespace ExitStatus
```

Comment:
Not `enum class`, to avoid casting to `int`? Interesting.

Reply:
But I _want_ it to be `int`, which is the type used for the exit status...